integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -445,13 +447,11 @@ case class CarbonLoadDataCommand(
       case c: CatalogRelation => c
     }.head.asInstanceOf[LogicalPlan]
 
+
     val query: LogicalPlan = if (dataFrame.isDefined) {
-      val timeStampformatString = CarbonProperties.getInstance()
-        .getProperty(CarbonCommonConstants.CARBON_TIMESTAMP_FORMAT,
-          CarbonCommonConstants.CARBON_TIMESTAMP_DEFAULT_FORMAT)
+      var timeStampformatString = CarbonCommonConstants.CARBON_TIMESTAMP_DEFAULT_FORMAT
       val timeStampFormat = new SimpleDateFormat(timeStampformatString)
-      val dateFormatString = CarbonProperties.getInstance().getProperty(CarbonCommonConstants
-        .CARBON_DATE_FORMAT, CarbonCommonConstants.CARBON_DATE_DEFAULT_FORMAT)
+      var dateFormatString = CarbonCommonConstants.CARBON_DATE_DEFAULT_FORMAT
       val dateFormat = new SimpleDateFormat(dateFormatString)
```

Comment:
move `timeStampformatString`, `timeStampFormat`, `dateFormatString`, `dateFormat` before line 451, they used in both if and else branch

Reply:
No, the logic is different. Both use different formats